commit 90dd51743200055f30d9e0e0337173118b4ae756
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Thu Oct 11 17:57:10 2012 +0300

    Fix noarch __isa_* macro filter in installplatform (RhBug:865436)
    
    - The filter wasn't doing what it was supposed to due to extra single
      quotes getting inserted, causing "rpmbuild --target noarch foo.spec"
      to whine about empty macro bodies. This is a regression introduced
      in rpm 4.10, commit 07ec480c180e4005a629242b8f9f8ab640e3e950 to be
      precise.

diff --git a/installplatform b/installplatform
index f7ae241..a68b3c0 100755
--- a/installplatform
+++ b/installplatform
@@ -104,7 +104,7 @@ for ARCH in noarch `grep ^arch_canon $RPMRC | cut -d: -f2`; do
     noarch)
 	CANONARCH=noarch
 	CANONCOLOR=0
-	FILTER="grep -v -E '^(%optflag|%__isa)'"
+	FILTER="grep -v -E ^(%optflag|%__isa)"
 	;;
   esac
 
